web/app/i18n/fr.json

```diff
@@ -0,0 +1,321 @@
+        "END_BEFORE_START": "La date de fin ne peut pas être antérieure à la date de début.",
+        "ERROR": "Erreur",
+        "EXPORT_INITIALIZATION_ERROR": "Erreur lors de l'initialisation de l'exportation.",
+        "EXPORT_TIMED_OUT": "La demande d'exportation a expiré.",
+        "FILTER_NOT_SAVED": "Filtre non enregistré",
+        "FUTURE_DATES": "La date et l'heure doivent être dans le passé.",
+        "INVALID_RANGE": "Le maximum doit être au moins aussi grand que la valeur minimale",
+        "NO_ACCESS": "L'utilisateur actuel n'a pas accès aux enregistrements",
+        "NO_AUDIT_RECORDS": "Aucun enregistrement d'audit n'existe pour le mois sélectionné",
+        "NO_RENDERER": "Aucun moteur de rendu défini pour le type de champ",
+        "RECORD_NOT_SAVED": "Enregistrement non enregistré",
+        "RECORD_SCHEMA_LOAD": "Impossible de télécharger le schéma d'enregistrement.",
+        "SAVING_FILTER_ERROR": "Erreur d'enregistrement du filtre",
+        "UNKNOWN_ERROR": "Erreur inconnue",
+        "UNKNOWN_RULE_TYPE": "Type de règle inconnu",
+        "VALUE_REQUIRED": "Valeur requise"
+    },
+    "LOGIN": {
+        "LOG_IN_TO_DRIVER": "Connectez-vous au DRIVER",
+        "LOG_IN_WITH": "Connectez-vous avec",
+        "PASSWORD": "Mot de passe",
+        "SIGN_IN": "Se connecter",
+        "USERNAME": "Nom d'utilisateur"
+    },
+    "MANAGEMENT": {
+        "ACCOUNT_INFORMATION": "Renseignements concernant le compte",
+        "ADMINISTRATOR?": "Administrateur?",
+        "ARCHIVE_NOTE": "L'autre enregistrement sera conservé pour l'avenir.",
+        "AUDIT_LOGS": "Journaux d'audit mensuels",
+        "EMAIL": "Email",
+        "GROUPS": "Groupes",
+        "KEEP_BOTH_RECORDS": "Conservez tous les deux enregistrements uniques",
+        "POTENTIAL_DUPLICATES": "Enregistrements en double potentiels",
+        "REALLY_ARCHIVE": "Êtes-vous sûr de vouloir utiliser cet enregistrement et d’archiver les données de l’autre enregistrement?",
+        "RESOLVE": "Résoudre",
+        "RESOLVE_DUPLICATE_ENTRIES": "Résoudre les enregistrements en double",
+        "TOKEN": "Token",
+        "USE_THIS_RECORD": "Utilisez cet enregistrement",
+        "USER_ID": "Identifiant d'utilisateur"
+    },
+    "MAP": {
+        "BLACKSPOT": "Point noir",
+        "BLACKSPOTS": "Points noirs",
+        "CDB_ATTRIBUTION": "!&copy; <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap</a> contributors, &copy; <a href='https://cartodb.com/attributions'>CartoDB</a>",
+        "ESRI_ATTRIBUTION": "!Tiles &copy; Esri &mdash; Source: Esri, i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community",
+        "HEATMAP": "Carte thermique",
+        "NUM_SEVERE": "Nombre d'incidents graves",
+        "SEVERITY_SCORE": "Score de gravité",
+        "STREETS": "Rues",
+        "SATELLITE": "Satellite"
+    },
+    "MONTH": {
+        "JANUARY": "janvier",
```

Comment:
These months should all be capitalized.

Reply:
These were provided this way. I think in general the French don't capitalize months or days of the week. Sent some examples to Rachel to clarify.

Reply:
Got input from Rachel so I updated to capitalize both months and days.